src/cascadia/TerminalApp/AboutDialog.cpp

```diff
@@ -86,16 +86,54 @@ namespace winrt::TerminalApp::implementation
             co_await wil::resume_foreground(strongThis->Dispatcher());
             UpdatesAvailable(true);
 #else // release build, likely has a store context
-            if (auto storeContext{ winrt::Windows::Services::Store::StoreContext::GetDefault() })
+            bool packageManagerAnswered{ false };
+
+            try
+            {
+                if (auto currentPackage{ winrt::Windows::ApplicationModel::Package::Current() })
+                {
+                    // We need to look up our package in the Package Manager; we cannot use Current
+                    winrt::Windows::Management::Deployment::PackageManager pm;
+                    if (auto lookedUpPackage{ pm.FindPackageForUser(winrt::hstring{}, currentPackage.Id().FullName()) })
+                    {
+                        using winrt::Windows::ApplicationModel::PackageUpdateAvailability;
+                        auto availabilityResult = co_await lookedUpPackage.CheckUpdateAvailabilityAsync();
```

Comment:
may want `if (auto availabilityResult = co_await lookedUpPackage.CheckUpdateAvailabilityAsync()) {...}`

cause you know, sometimes these APIs just decide to return null

Reply:
Call me optimistic. Watch this come back to bite me later. :)